### PR TITLE
Friends to follower

### PR DESCRIPTION
**Pull request by Nokius**
As in #51 mention the User can see Friends.
I assume it should be Follower.

### CODE REVIEW

File: src/bin/qml/pages/RightPanel.qml [hunk 1]
```diff
@@ -84,7 +84,7 @@ Item {
     function openFriends(userId, screenName, account, pushMode) {
         var args = {"user_id": userId}
         var params = {
-            title: qsTr("%1's friends").arg(screenName),
+            title: qsTr("%1's follower").arg(screenName),
```
Comment (SfietKonstantin) [inline src/bin/qml/pages/RightPanel.qml]: followers ? you usually don't have only one follower :)

File: src/bin/qml/pages/UserPage.qml [hunk 2]
```diff
@@ -145,7 +145,7 @@ Page {
                 }
                 MiniButton {
                     width: parent.width / 3
-                    text: query.user ? qsTr("%n\nfriends", "", query.user.friendsCount) : ""
+                    text: query.user ? qsTr("%n\nfollower", "", query.user.friendsCount) : ""
                     onClicked: container.panel.openFriends(query.userId, query.user.screenName,
```
Comment (SfietKonstantin) [inline src/bin/qml/pages/UserPage.qml:149]: Same

File: src/bin/qml/pages/RightPanel.qml (hunk 1 above)
Comment (SfietKonstantin) [inline src/bin/qml/pages/RightPanel.qml]: Damn, it's not that, thanks for the heads-up:
It should be something like: "%1 is following"

File: src/bin/qml/pages/UsersPage.qml [hunk 3]
```diff
@@ -108,7 +108,7 @@ Page {
 
         StatusPlaceholder {
             model: userModel
-            errorMessage: (model.status === Model.Idle && model.count === 0) ? qsTr("No friends") : ""
+            errorMessage: (model.status === Model.Idle && model.count === 0) ? qsTr("No follower") : ""
```
Comment (Nokius) [inline src/bin/qml/pages/UsersPage.qml]: follower or followings 
I was not able to see it in the UI where it pops up :-S

Comment (SfietKonstantin) [inline src/bin/qml/pages/UsersPage.qml]: I think this is a bug here. I'm using the same page for different usages

Comment (SfietKonstantin) [inline src/bin/qml/pages/UsersPage.qml]: You know what, write "No user" here

Comment (Nokius) [inline src/bin/qml/pages/UsersPage.qml]: :)

**Comment by SfietKonstantin** [comment]
Squash commits and LGTM

**Comment by Nokius** [comment]
not sure if I did the squash correct (first time ever) Thanks for this lesson 
here is the squash PR
#53

**Pull request closed without merging**